src/main/java/cn/nukkit/command/Command.java

```diff
@@ -135,7 +136,7 @@ public Map<String, CommandOverload> getOverloads() {
         return this.commandData.overloads;
     }
 
-    public abstract boolean execute(CommandSender sender, String commandLabel, String[] args);
+    public abstract boolean execute(@NotNull CommandSender sender, String commandLabel, String[] args);
```

Comment:
aren't `commandLabel` and `args` non-nullable as well?

Reply:
You right, i missed this moment